CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# v2.1.0 (2024-07-08)
+
+## OS Changes
+* Update kernels: 5.10.218-208, 5.10.219, 5.15.160-104, and 6.1.94 ([#13], [#17])
+* Include kmod-6.1-neuron package in core kit ([#21])
```

Comment:
nit: Include is a useful word when describing variants, for the core kit, I personally prefer "Add" vs "Include". 
```suggestion
* Add kmod-6.1-neuron package to core kit ([#21])
```